far/FarEng.hlf.m4

```diff
@@ -6179,144 +6179,167 @@ copied.
 
- Изменение этого параметра возможно через ~far:config~@FarConfig@
+ This parameter can be changed via ~far:config~@FarConfig@ only.
 
 
 @XLat.WordDivForXlat
 $ #far:config XLat.WordDivForXlat#
- Параметр "XLat.WordDivForXlat" позволяет задавать используемые в функции транслитерации (XLat,
-для преобразования текущего слова без выделения) символы-разделители слов.
+ This string parameter defines word delimiters for transliteration
+(#XLat# function) of the current word without selecting it.
 
- По умолчанию: пробел табуляция и символы #!##$%^&*()+|=\\/@?#
+ Default value: #Space# (U+0020), #Tab# (U+0009) and characters
+#!##$%^&*()+|=\\/@?#.
 
- Изменение этого параметра возможно через ~far:config~@FarConfig@
+ This parameter can be changed via ~far:config~@FarConfig@ only.
 
 
 @Editor.ReadOnlyLock
 $ #far:config Editor.ReadOnlyLock#
- Параметр "Editor.ReadOnlyLock" задаёт поведение редактора при открытии файлов с атрибутами ReadOnly, Hidden и System.
+ This numeric parameter controls the behavior of the Editor when opening
+files with #Read-only#, #Hidden# or #System# attributes.
 
- Биты:
- 0 - Блокировать редактирование файла с атрибутом R/O
- 1 - Предупреждать при открытии файла с атрибутом R/O
- 2 - не используется
- 3 - не используется
- 4 - не используется
- 5 - применять дополнительно для файлов с атрибутом Hidden
- 6 - применять дополнительно для файлов с атрибутом System
-
- Значения в битах 0 и 1 соответствуют опциям в диалоге ~настроек редактора~@EditorSettings@  ("Блокировать
-редактирование файла с атрибутом R/O" и "Предупреждать при открытии файла с атрибутом R/O").
+ Bit numbers:
+ 0 - ^<wrap>Lock down editing of read-only files;
+ 1 - Warn when opening read-only files;
+ 2 - Unused;
+ 3 - Unused;
+ 4 - Unused;
+ 5 - Also, apply the behavior defined by bits 0 and 1 to hidden files;
+ 6 - Also, apply the behavior defined by bits 0 and 1 to system files.
 
- Например, значение 0x43 - предупреждать и блокировать изменения в файлах с атрибутами ReadOnly и System.
+ Default value: 0 (allow editing of any files without warnings).
 
- По умолчанию значение = 0.
+ For example, if this parameter is set to #0x43# (binary 0100'0011),
+warning will be shown when opening read-only and system files; editing
+of such files will be disabled.
 
- Изменение этого параметра возможно через ~far:config~@FarConfig@
+ This parameter can be changed via ~far:config~@FarConfig@. Bits 0 and
+1 are also controlled by the options #Lock editing of read-only files#
+and #Warn when opening read-only files# of the ~Editor~@EditorSettings@
+settings dialog.
 
 
 @Editor.FileSizeLimit
 $ #far:config Editor.FileSizeLimit#
- Параметр "Editor.FileSizeLimit" задаёт максимальный размер редактируемого файла в байтах.
-Если размер редактируемого файла превышает максимально допустимый, то будет показано предупреждающее сообщение перед
-открытием такого файла.
+ This numeric parameter defines the maximum size of a file in bytes
+which can be safely loaded into Editor.
```

Comment:
"Safely" sounds a little alarming, like "something bad is going to happen if I proceed".
Maybe something like "reasonable size"?

It's probably worth explaining why this can be useful in the first place, e.g. "since the whole content of a file is read into memory, accidental opening of very large files could be undesirable".

Reply:
Yeah, probably. I was not happy with this article as well. Thank you for pointing out. I reworked it entirely, both English and Russian. See if you like it this way.